Doc/faq/design.rst

```diff
@@ -422,8 +422,10 @@ two to the directory.
 
 Tuples are immutable, meaning that once a tuple has been created, you can't
 replace any of its elements with a new value.  Lists are mutable, meaning that
-you can always change a list's elements.  Only immutable elements can be used as
-dictionary keys, and hence only tuples and not lists can be used as keys.
+you can always change a list's elements.  Only :term:`hashable` objects can
```

Comment:
I think you can reference all three terms (immutable, mutable and hashable), when they first appear in this paragraph.